src/registrar/templates/domain_dsdata.html

```diff
@@ -42,7 +42,7 @@
     {% include "includes/form_errors.html" with form=form %}
   {% endfor %}
 
-  <h1>DS data</h1>
+  <h1 id="domain-dsdata">DS data</h1>
```

Comment:
Same as above, we already have unique ids introduced on the forms below

Reply:
^^^